packages/docusaurus-module-type-aliases/src/index.d.ts

```diff
@@ -346,3 +346,10 @@ declare module '*.css' {
   const src: string;
   export default src;
 }
+
+interface Window {
+  docusaurus: {
+    prefetch: (url: string) => false | Promise<void[]>;
+    preload: (url: string) => false | Promise<void[]>;
```

Comment:
Nit but what's the advantage of having `void[]` here?

Maybe it would be better to use `await Promise.all([])` and just have `Promise<void>`?

Reply:
Ahhh... you are right. Not a huge problem though.

Reply:
Ah, wait, we can't use `await`. We never use async-await in client code because that bloats the bundle (in fact I also want to reduce `??` and `?.` usages). So in order to make it return `void`, we have to chain another `.then(() = {})` which is not really ideal.

In addition, when it can't prefetch, `false` is returned synchronously, instead of being wrapped in a promise.